taichi/system/traceback.h

```diff
@@ -0,0 +1,6 @@
+#include "taichi/common/core.h"
+
+namespace taichi {
```

Comment:
Shall we write `namespace taichi` as-is or use this one?
https://github.com/taichi-dev/taichi/blob/dee35f4830a130ceffeeb0d1fdf377d8ca9c7124/taichi/common/core.h#L154

Reply:
Yeah, I think both are fine. At some point we should do a batch change from `TI_NAMESPACE_BEGIN` to `namespace taichi {` anyway..

Reply:
Yeah I also prefer `namespace taichi {`... It took me one more click in the IDE to know what `TI_NAMESPACE_BEGIN` was.